server/src/mindustry/server/ServerControl.java

```diff
@@ -172,6 +172,18 @@ public ServerControl(String[] args){
             }
         });
 
+        Events.on(WaveEvent.class, e -> {
+            if(state.is(State.playing) && Config.valueOf("enableAutosave").bool() ){
```

Comment:
Why do `Config.valueOf("enableAutosave")` when you can do `Config.enableAutosave`? You also have an unnecessary spaces after `bool()`.

Reply:
Copy pasted the wrong code apparently.